Index: src/main/java/com/java/demo/c/c/controller/HelloController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/java/demo/c/c/controller/HelloController.java	(date 1578275013000)
+++ src/main/java/com/java/demo/c/c/controller/HelloController.java	(date 1578292912844)
@@ -11,5 +11,6 @@
     public String Hello(@RequestParam(name = "name") String name , Model model){
         model.addAttribute("name" , name);
         return "Hello";
+
     }
 }
